packages/scrabble-solver/src/lib/detectLocale.ts

```diff
@@ -9,6 +9,11 @@ const detectLocale = (): Locale => {
     return Locale.EN_GB;
   }
 
+  //TODO not sure if this is needed
+  if (window.navigator.languages.includes('fa-FA')) {
```

Comment:
Yes, it is needed to detect the language when user opens the app for the first time.

```suggestion
  if (window.navigator.languages.includes('fa') || window.navigator.languages.includes('fa-IR')) {
```